build.gradle

```diff
@@ -43,6 +43,7 @@ dependencies {
     implementation 'com.netflix.nebula:gradle-contacts-plugin:latest.release'
     implementation 'org.eclipse.jgit:org.eclipse.jgit:5.7.0.202003110725-r'
     implementation 'org.tmatesoft.svnkit:svnkit:1.8.12'
+    implementation 'net.java.dev.jna:jna-platform:5.7.0'
```

Comment:
It was being pulled in transitively by svnkit before, but was used directly in the project.